test/src/KeyResetMessageTest.cc

```diff
@@ -549,7 +549,7 @@ TEST_F(KeyResetMessageTest, check_receive_message_to_revoked_key_from_unknown) {
     free_identity(from_ident);
 }
 
-
+#if 0
```

Comment:
Please remove tests that are completely obsolete, that is, they can't be adapted or it doesn't make sense. No `#if 0`.